pyvista/plotting/qt_plotting.py

```diff
@@ -634,10 +678,14 @@ class BackgroundPlotter(QtInteractor):
     ICON_TIME_STEP = 5.0
 
     def __init__(self, show=True, app=None, window_size=None,
-                 off_screen=None, allow_quit_keypress=True, **kwargs):
+                 off_screen=None, allow_quit_keypress=True,
+                 menu_bar=True, **kwargs):
         """Initialize the qt plotter."""
         if not has_pyqt:
             raise AssertionError('Requires PyQt5')
+        if not isinstance(menu_bar, bool):
+            raise TypeError("Expected type for ``menu_bar`` is bool"
+                            " but {} was given.".format(type(menu_bar)))
```

Comment:
Any specific reason to check this instead of just duck-typing?

Reply:
Because I was not lazy enough. But at least, I have my peace of mind after this check. If someone has a good reason, I think they can remove it. Meanwhile I think it does not hurt so much.

Reply:
I think @imsodin has a point since we're not doing type checking on other inputs.  We either should be consistent with our type checking, or simply duck-type `menu_bar` and `tool_bar`.  However, I'll leave it up to you as it doesn't hurt anything by having these type checks.